src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { FC, useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { Link, useLocation } from 'react-router-dom';
+
+import MLHBANNER from '../../assets/images/mlh-badge.svg';
+import './styles.scss';
+
+const Navbar: FC<{ backgroundColor?: string; textColor?: string; breakLineColor?: string; showBanner?: Boolean }> = ({
+    backgroundColor,
+    textColor,
+    breakLineColor,
+    showBanner = true,
+}): JSX.Element => {
+    let { pathname } = useLocation();
+    const [openDrawer, toggleDrawer] = useState(false);
+    const drawerRef = useRef(null);
+
+    useEffect(() => {
+        const closeDrawer = (event: { target: any }) => {
```

Comment:
`MouseEvent` btw

Reply:
tyty, but I feel like there was a reason why I did not put it